cfgov/jinja2/v1/_includes/molecules/expandable.html

```diff
@@ -40,12 +40,11 @@
     %}
 {% endif %}
 {% set value = value or global_dict.expandable %}
-
 <div class="m-expandable
-            m-expandable__expanded
```

Comment:
So this'll make it so when JS is off they don't have the `m-expandable__expanded` modifier, yet they'll be expanded. So for example, they'll all be expanded, but the markup won't be equal across all of them, which shouldn't be the case:

![screen shot 2016-01-04 at 10 28 29 am](https://cloud.githubusercontent.com/assets/704760/12092908/f8e29c66-b2cd-11e5-9224-b5bd71addb15.png)